common/lifecyleMgr.go

```diff
@@ -276,6 +278,12 @@ func (lcm *lifecycleMgr) Info(msg string) {
 }
 
 func (lcm *lifecycleMgr) Prompt(message string, details PromptDetails) ResponseOption {
+
+	if shouldQuietMessage(outputMessage{msgType: eOutputMessageType.Prompt()}, lcm.OutputVerbosityType) {
```

Comment:
Why is this required after you've made sure conflicting options are not allowed?

Reply:
Conflicting options are not allowed = the user can't pass in quiet or essential mode with overwrite=prompt.
What I'm doing here is checking if the current output-level is quiet or essential. If so, I'm passing yes as the answer to all prompts. This will not be the case when output-level=default.
This is logically the same as "if output-level is not 'default', return yes as the answer to the prompt"

Reply:
closing this now

Reply:
Unresolving. This is unreachable code.

Reply:
makes sense, changing this.